ui/src/shared/components/AutoRefresh.js

```diff
@@ -114,13 +116,25 @@ const AutoRefresh = ComposedComponent => {
 
       const timeSeriesPromises = queries.map(query => {
         const {host, database, rp} = query
+
+        const templatesWithResolution = templates.map((temp) => {
+          if (temp.tempVar === ':autoGroupBy:') {
+            if (resolution) {
+              return {...temp, resolution}
+            }
+            return {...temp, resolution: 1000}
```

Comment:
Though I just noticed `DashboardPage.js` L266 -- can't we assume that it will exist?